platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+SSD_FW_UPGRADE="/host/ssd_fw_upgrade"
+
+if [ -e $SSD_FW_UPGRADE/GPIO7_high ]; then
+    systemctl start --no-block s6100-ssd-monitor.timer
```

Comment:
why do you need to start timer here again?

Reply:
As discussed, this timer needs to be started only on two cases - when SSD FW is being upgraded first time and then on reboots when GPIO7 status is still high. This belongs to second case.